pkg/relayer/session/session.go

```diff
@@ -104,7 +120,13 @@ func NewRelayerSessions(
 		opt(rs)
 	}
 
-	if err := rs.validateConfig(); err != nil {
+	if err = rs.validateConfig(); err != nil {
+		return nil, err
+	}
+
+	// Initialize the session metadata store.
+	sessionRegistryDir := path.Join(rs.storesDirectory, "sessions_metadata")
```

Comment:
Align naming once we align on something. First time I'm seeing `sessionRegistryDir`.